pages/linux/rmlint.md

```diff
@@ -0,0 +1,20 @@
+# rmlint
+
+> Extremely fast tool to remove remove duplicates and other lint from your filesystem.
```

Comment:
It's better to just stick to the facts, and avoid adding assessments of speed, quality, aesthetics, etc. Also, note that you wrote "remove" twice :) Here's a suggestion for this description:

```suggestion
> Identify duplicate files or directories, and other filesystem issues.
```

Reply:
Applied in [`d53236f`](https://github.com/tldr-pages/tldr/pull/9090/commits/d53236f122d503f3fb8284f6821bfd4707421a15).